wiki/Mapping_Techniques/Inspiration_Gallery_Corner/en.md

```diff
@@ -0,0 +1,176 @@
+# Inspiration Gallery Corner
```

Comment:
not really, inconsistent screenshots 👎 this was emerged from `Making Good Sliders`... delete?